src/zenml/cli/artifact.py

```diff
@@ -166,7 +175,7 @@ def prune_artifacts(
     for unused_artifact in unused_artifacts:
         try:
             Client().delete_artifact(
-                artifact_id=unused_artifact.id,
+                name_id_or_prefix=unused_artifact.id,
```

Comment:
It will only wipe our latest versions of them or?

Reply:
Reworked that part heavily